apps/fluent-tester/src/FluentTester/FluentTester.tsx

```diff
@@ -128,10 +128,8 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
   const TestComponent = selectedTestIndex == -1 ? EmptyComponent : sortedTestComponents[selectedTestIndex].component;
 
   const TestListSeparator = Separator.customize({
-    tokens: {
-      color: useTheme().colors.inputBorder,
-      separatorWidth: 2,
-    },
+    color: useTheme().colors.inputBorder,
+    separatorWidth: 2,
```

Comment:
I know you didn't write this, but its a little weird to be doing this in a render function :)

Reply:
Ugh, didn't notice this, I will fix it as this is wrong and I'm surprised it isn't crashing

Reply:
I guess it won't crash because useContext isn't a memo based hook, so it doesn't create storage. This gets called out of tree though, which means it will always use the initialized value, which is undefined, so I would expect that this always ends up hitting the fallback theme.